ext/hamlit/hamlit.c

```diff
@@ -1,5 +1,6 @@
 #include <ruby.h>
 #include <ruby/encoding.h>
+#ifndef TRUFFLERUBY
```

Comment:
This seems to skip this entire file under file under TruffleRuby, is it because of missing `rb_ary_sort_bang`?
Is a pure-Ruby implementation used instead then?

Reply:
Both are correct. https://github.com/oracle/truffleruby/issues/1600 was also relevant, but I could workaround it by casting `VALUE` to `ptrdiff_t`.